app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,23 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(params.require(:comment).permit(:text, :user_id, :post_id))
```

Comment:
- [ ] Hey! 👋 Take a peek at this action controller. 👀 You've already set up params in the private method—fantastic! Now, let's simplify things by calling comments_param within the new method of the comment create controller. 🔄 This ensures a cleaner structure and follows best practices. Keep up the good work! 💪🚀

```suggestion
   @comment = @current_user.comments.new(params)
    @comment.post_id = params[:post_id]
```